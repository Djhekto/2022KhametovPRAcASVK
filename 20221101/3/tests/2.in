a, b = Grange(1, 2, 2049), Grange(1, 3, 1)
print(f"{len(a)=}, {len(b)=}")
print(f"{list(a)}({not(a)}), {list(b)}({not(b)})")
print(f"{a=}, {b=}")
print(f"{len(list(a))=}, {len(list(b))=}")
print(f"{a[3]=}, {a[13]=}")
a, b = a[1024:4097], a[1024:8193:2]
print(f"{a=}, {a[5]=}")
print(f"{b=}, {b[5]=}, {list(b)=}")