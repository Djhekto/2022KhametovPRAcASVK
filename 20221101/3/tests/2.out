len(a)=12, len(b)=0
[1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048](False), [](True)
a=grange(1, 2, 2049), b=grange(1, 3, 1)
len(list(a))=12, len(list(b))=0
a[3]=8, a[13]=8192
a=grange(1024, 2, 4097), a[5]=32768
b=grange(1024, 4, 8193), b[5]=1048576, list(b)=[1024, 4096]
